mtest/musicxml/io/testColors.xml

```diff
@@ -103,6 +103,7 @@
         <notations>
           <articulations>
             <strong-accent type="up" color="#C3D610"/>
+            <breath-mark color="#00FFFF" placement="above">comma</breath-mark>
```

Comment:
@rettinghaus any idea why in my backport that placement is needed, but wasn't in your PR?

Reply:
I guess it has something to do with placment handling in `addArticulationToChord`. Some strange things are going on, I couldn't yet pinpoint. I tried to add `ArticulationAnchor::AUTO` as default, but that lead to an other failed test.

Reply:
Hmm, might be due to a much earlier change in the export code, which compensates for the old import code to default breaths to below, the new defaults to above, so that compensation on export to force it above isn't needed anymore

And a missing `else` is the reason why I needed that other mtest change

Reply:
See https://github.com/musescore/MuseScore/blob/7d370648ed77f21339ef906ec5bb55f8a11bcd79/src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp#L3446-L3448

Reply:
So I guess for now I'd have to live with that difference